src/System.IO.FileSystem/tests/DirectoryInfo/GetSetAttributes.cs

```diff
@@ -127,5 +127,39 @@ public void WindowsInvalidAttributes_ThrowArgumentException(FileAttributes attr)
             test.Create();
             Assert.Throws<ArgumentException>(() => Set(test.FullName, attr));
         }
+
+        [ConditionalFact(nameof(CanCreateSymbolicLinks))]
+        public void SymLinksAreReparsePoints()
+        {
+            var path = GetTestFilePath();
+            var linkPath = GetTestFilePath();
+
+            Directory.CreateDirectory(path);
+            Assert.True(MountHelper.CreateSymbolicLink(linkPath, path, isDirectory: true));
+
+            Assert.NotEqual(FileAttributes.ReparsePoint, FileAttributes.ReparsePoint & Get(path));
+            Assert.Equal(FileAttributes.ReparsePoint, FileAttributes.ReparsePoint & Get(linkPath));
+        }
+
+        [ConditionalFact(nameof(CanCreateSymbolicLinks))]
+        public void SymLinksReflectSymLinkAttributes()
```

Comment:
Yeah, this was my quick attempt to say that concisely... I guess I could say "TheirOwn" or something like that. I'll change it if I have other reasons to resubmit.